### ISSUE TITLE
support for  classpath 'com.google.gms:google-services:4.3.3'

### ISSUE THREAD

**Issue by pietromanneschi** [chemerisuk/cordova-support-google-services#35]
Hello,

is it possible to update to 
classpath 'com.google.gms:google-services:4.3.3' ?

Would there be any downsides?

**Comment by chemerisuk** [comment]
@pietromanneschi not sure... Why do you need this particular version?

**Comment by pietromanneschi** [comment]
I had a conflict when using the codepush plugin 
https://github.com/microsoft/cordova-plugin-code-push 
and another plugin that actually i don't remember. This one i think
https://github.com/phonegap/phonegap-plugin-push

They both target google-services class, but with different version, 4.2.0 and 4.1.0 if i remember correctly, and i read that there is a problem in the version checker, as you see in this comment
https://github.com/microsoft/react-native-code-push/issues/1448#issuecomment-469902328
that reference this other.
https://github.com/google/play-services-plugins/issues/30

I modified your plugin locally to target 4.3.3 and solved the problem. Another workaround was to add 
com.google.gms.googleservices.GoogleServicesPlugin.config.disableVersionCheck = true

at the bottom of android/app/build.gradle

**Comment by mingyiyangyarris** [comment]
Hey @chemerisuk, I'm facing the same issue as pietromanneschi did. 

Do you mind merging PR #36 to upgrade google service version?

**Comment by wilsolutions** [comment]
Same here...

**Comment by Sperows** [comment]
I have issue with this:
> Configure project :app
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)

After setting version to 4.3.3 the warning goes away. Unfortunately, I have to set it every time after I add android platform.

**Comment by pietromanneschi** [comment]
@Sperows you can change it in your local folder to avoid changing it everytime you remove/add the platform. In the root of your cordova project, you have a folder "plugins", with the folder "cordova-support-google-services". there you find the build.gradle. Change it in there

**Comment by Sperows** [comment]
> @Sperows you can change it in your local folder to avoid changing it everytime you remove/add the platform. In the root of your cordova project, you have a folder "plugins", with the folder "cordova-support-google-services". there you find the build.gradle. Change it in there

Oh, nice one - thanks for the tip! I do apologize for this noobish mistake :) I have just started with cordova and android/ios deployments and it's all new to me.

**Comment by chemerisuk** [comment]
The version upgrade published in v1.4.0. All my firebase plugins use 
```xml
<dependency id="cordova-support-google-services" version="^1.3.0"/>
```
Therefore updated version of `cordova-support-google-services` will be fetched automatically on plugin reinstall.
